cmd/icinga-notifications-daemon/main.go

```diff
@@ -15,10 +16,17 @@ import (
 
 func main() {
 	var configPath string
+	var version bool
```

Comment:
I'd rename the variable (and just the variable) to `showVersion`. Better expresses what the value means.